tests/TestCase/Mailer/EmailTest.php

```diff
@@ -2712,6 +2712,7 @@ public function testJsonSerialize()
         ];
         $this->assertEquals($expected, $result);
 
+        debug(unserialize(serialize($this->CakeEmail)));
         $result = json_decode(json_encode(unserialize(serialize($this->CakeEmail))), true);
```

Comment:
Nope it won't. As I said in my initial comment i don't know how to deal with this.